core/src/main/scala/org/genivi/sota/core/VehicleUpdatesResource.scala

```diff
@@ -174,8 +185,10 @@ class VehicleUpdatesResource(db : Database, resolverClient: ExternalResolverClie
   val route = {
     (pathPrefix("api" / "v1" / "vehicle_updates") & extractVin) { vin =>
       get {
-        pathEnd { logVehicleSeen(vin) { pendingPackages(vin) } } ~
-        path("queued") { pendingPackages(vin) } ~
+        namespaceExtractor { ns =>
```

Comment:
We don't need a namespace. at the moment we need to assume all vins are unique, so we should not use namespaces for these calls when we don't have one.

This is trickier that that, because the `namespaceExtractor` will not fail, it will return a namespace scoped to that vehicle, that is what auth+ is returning, so `namespaceExtractor` will happily pass a wrong namespace to these methods, which will return unpredictable results using a namespace different from the one the vehicle was created with.